Cargo.lock

```diff
@@ -1083,6 +1083,7 @@ dependencies = [
  "sway-types",
  "sway-utils",
  "term-table",
+ "terminal-link",
```

Comment:
revert changes to lockfile

Reply:
Yep, will revert, sorry, must have snuck in from previous PR.